src/state/order_schema/order_packet.rs

```diff
@@ -41,6 +41,12 @@ pub enum OrderPacket {
         /// Using only deposited funds will allow the trader to pass in less accounts per instruction and
         /// save transaction space as well as compute. This is only for traders who have a seat
         use_only_deposited_funds: bool,
+
+        /// If this is set, the order will be invalid after the specified slot
+        last_valid_slot: Option<u64>,
+
+        /// If this is set, the order will be invalid after the specified unix timestamp
+        last_valid_unix_timestamp_in_seconds: Option<u64>,
```

Comment:
Note: By changing this schema, there will likely need to be special logic built into the indexer (down the line) to handle the differences between the most up to date version of the order packet and an older version (backfills for transactions prior to this change that look at instruction data)

Reply:
Maybe it makes sense to make a new order type altogether called TimeInForce? This feels less breaking to me.

Every other change (beside the CondensedOrder) doesn't expose a breaking state change to contract interaction. New order type allows existing systems to carry on

Reply:
don't you need to implement all the other order types within TIF? limit and post-only at least?